vector-im/element-android
# Setup Flipper

## Type of change


## Content


- Setup flipper on the project, for debug builds only (https://fbflipper.com/)
- Move getLastSession() to the SessionManager
- Create `DebugService` on the SDK to get access to the `Realmconfiguration`s
- Move `logDbUsageInfo()` to `DebugService`

What will be done later:
- write documentation (the file is here, but quite empty). This PR is created to unblock others dev faster.

## Motivation and context



Help developer to develop the app, especially by inspecting the databases content.

Closes #6305 

## Screenshots / GIFs

### Network

<img width="1723" alt="image" src="https://user-images.githubusercontent.com/3940906/173559677-509b5233-f65c-46cb-99fb-3d8541266156.png">

### Database

<img width="1543" alt="image" src="https://user-images.githubusercontent.com/3940906/173559782-b7303f60-c5b9-4272-9c2a-3d86bc092dfe.png">

### LayoutInspector

<img width="1720" alt="image" src="https://user-images.githubusercontent.com/3940906/173560194-5ccf7b85-0db4-459c-bfca-09b604c257ba.png">

### On phone Diagnostic Activity

<img width="250" alt="image" src="https://user-images.githubusercontent.com/3940906/173561049-2638364e-256c-4031-9b77-0606dc10db6d.png">




## Tests



- Run the app in debug and use Flipper app
- Build the app in release mode, to ensure that it compiles.

## Tested devices

- OS version(s):

## Checklist
